app/components/footer/talk_to_us_component.html.erb

```diff
@@ -17,14 +17,15 @@
                 <p class="telephone">0800 389 2500</p>
               </div>
             </div>
-            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
+            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong>
+            Please note that the contact centre will be shut on bank holidays over the Christmas period, on Monday 27th December, Tuesday 28th December, and Monday 3rd January. There will also be an early close of 1pm on Christmas Eve. Responses may be slower during this period.</p>
```

Comment:
```suggestion
            Chat will be unavailable from 1pm on 24 December until 29 December. It will also be closed on 3 January. Responses may be slower over the Christmas period.</p>
```